test/webpack/karma.conf.js

```diff
@@ -64,7 +65,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
+    browsers: ['ChromeHeadless'],
```

Comment:
Not sure if use of PhantomJS is intentional or accidental. In case it's intentional I would love to learn the reasons. In case it was accidental - here is my rationale for switching.

[PhantomJS is suspended](https://github.com/ariya/phantomjs/issues/15344) for about 2 years already. When I was writing tests it was giving me all kinds of weird errors for `Uint8Array`. Probably the support there is limited. So I figured it's easier to switch to something more up-to-date and stable than figure out those errors.